samples/04_gis_analysts_data_scientists/classifying_human_activity_using _tabPFN_classifier.ipynb

```diff
@@ -0,0 +1,2179 @@
+{
```

Comment:
<p> </p><p>To get a sense of what the training data looks like, the&nbsp;<code>show_batch()</code>&nbsp;method will randomly pick a few training samples and visualize them. The samples show the explanatory variables and the&nbsp;<code>Activity</code>&nbsp;target label to predict.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2189/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>